signalflow/computation.go

```diff
@@ -352,6 +355,11 @@ func (c *Computation) Data() <-chan *messages.DataMessage {
 	return c.dataCh
 }
 
+// Events returns the results from events or alerts queries
+func (c *Computation) Events() []*messages.EventMessage {
+	return c.events
```

Comment:
This need to be synchronized and waited on like all the other methods that return execution metadata, such as https://github.com/signalfx/signalfx-go/blob/master/signalflow/computation.go#L117.